plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveOnDataLake.java

```diff
@@ -244,6 +254,731 @@ public void testFlushPartitionCache()
         computeActual(format("DROP TABLE %s", fullyQualifiedTestTableName));
     }
 
+    @Test
+    public void testPartitionProjectionOnVarCharColumnWithSingleEnumProjectionAndColumnNameWithWhitespace()
+    {
+        String tableName = "nation_" + randomTableSuffix();
+        String fullyQualifiedTestTableName = getTestTableName(tableName);
+
+        computeActual(getCreateTableStatement(
```

Comment:
It's best to avoid these types of helper utilities in test code, since it makes it hard to see what is being tested. Also, the test code itself isn't tested, so it's easy to make mistakes.

The goal of tests is to be easy to read and understand, so having the literal SQL in the tests is better.

Reply:
Refactored them to clear SQL.